src/View/Helper/BakeHelper.php

```diff
@@ -64,6 +65,7 @@ public function arrayProperty(string $name, array $value = [], array $options =
      * @param array $list array of items to be stringified
      * @param array $options options to use
      * @return string
+     * @deprecated 2.5.0 Use BakeHelper::exportVar() instead.
```

Comment:
Will it be hard to switch to exportVar()? I think we want to add deprecationWarning() at the same time in most cases.

Reply:
It would be actually easier to change all the funky string array to regular ones :)